### PR TITLE
EVW-1728: Check Your Answers Page

### PR DESCRIPTION
**Pull request by cksanders**
https://jira.digital.homeoffice.gov.uk/browse/EVW-1728 

includes:
- the feature Check Your Answers Page
- two bug fixes (arrival / departure dates/times)

Potential to refactor the AT's to remove duplication; possibly only until HOF is updated to jump into steps.

### CODE REVIEW

**Comment by roc** [comment]
Looks good!

Could you:
- add a screenshot
- rebase out the `WIP` commits [564139d](https://github.com/UKHomeOffice/evw-customer-hof/pull/35/commits/564139db362e0db6aca628954e3f0c436a604989) and [a71988c](https://github.com/UKHomeOffice/evw-customer-hof/pull/35/commits/a71988c2a11ee5ad8b12460ccd120542a9a83983)


See #3 #7 #9 #24  for examples

File: acceptance_tests/features/apply.feature [hunk 1]
```diff
@@ -1,86 +1,95 @@
 @apply
 Feature: Apply for a new EVW
 
-Scenario: The user chooses to apply for a new EVW
+  Scenario: The user chooses to apply for a new EVW
```
Comment (roc) [inline acceptance_tests/features/apply.feature]: Would it be okay to keep the indentation as is?

Comment (cksanders) [inline acceptance_tests/features/apply.feature]: I wouldn't suggest this ... Scenario is an element/child of Feature.

**Comment by cksanders** [comment]
@roc  ...

1) I will attached the screen shot.

2) "rebase out the WIP commits 564139d and a71988c"

"WIP"  or 564139d is a commit made by David: It is a work in process - meaning some of the functionality is not redundant, specifically the translations. Commit a71988c is only the removal of the redundant controller, hence the commit message/name "	EVW-1728: Check Your Answers Page - Remove WIP controller"

File: acceptance_tests/features/apply.feature (hunk 1 above)
Comment (cksanders) [inline acceptance_tests/features/apply.feature]: I have rebased the "WIP" commit and squashed it into the relevant commit.

**Comment by roc** [comment]
🎉 wicked🎉

**Pull request merged**